BLSQ/iaso
# IA-1884 Instance: save the panningId send by the mobile app

Mobile app was sending the planning id the submission was for but it was not saved server side.

thus we couldn't link back.
Fix IA-1884

## Self proof reading checklist

- [NA] Are my typescript files well typed
- [NA] Did I add translations

## Changes

Add field to add the info, add test

## How to test

Create a planning and send a submission from the app, check that we have the info in DB (it's not exposed in the API yet)
